extensions/sql-database-projects/.gitignore

```diff
@@ -0,0 +1 @@
+*.vsix
```

Comment:
You should add a vscodeignore as well so that unnecessary things aren't bundled into the produced VSIX (such as tsconfig.json, src files, etc)